ooc-opengl.use

```diff
@@ -14,7 +14,6 @@ Imports: OpenGLUv
 Imports: OpenGLCanvas
 Imports: OpenGLSurface
 Imports: OpenGLMap
-Imports: OpenGLMapPack
```

Comment:
If you don't mind, do that in another PR. I'm happy otherwise and almost done testing.